ember-cli-build.js

```diff
@@ -2,7 +2,7 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
+  let app = new EmberAddon(defaults, {
```

Comment:
block-scoped variables aren't supported on some of the node versions that we test with in the CI build 

```Could not require 'ember-cli-build.js': Block-scoped declarations (let, const, function, class) not yet supported outside strict mode```

Reply:
I removed Node 4 from `.travis.yml` - I couldn't see why we'd still have to run that.